app/src/main/java/hr/dice/filipbionda/tmdbpractice/ui/showscreen/ShowScreen.kt

```diff
@@ -0,0 +1,671 @@
+            )
+            Spacer(
+                modifier = Modifier.height(dimensionResource(R.dimen.show_screen_spacer_height_2))
+            )
+            ExpandedButton(
+                onClick = {
+                    playTrailer(mockMovie.trailerUrl)
+                },
+                modifier = Modifier.padding(horizontal = dimensionResource(R.dimen.show_screen_horizontal_padding)),
+                content = {
+                    Row {
+                        Icon(
+                            imageVector = Icons.Outlined.PlayCircle,
+                            contentDescription = null
+                        )
+                        Spacer(
+                            modifier = Modifier.width(dimensionResource(R.dimen.play_trailer_items_padding))
+                        )
+                        Text(
+                            text = stringResource(R.string.play_trailer)
+                        )
+                    }
+                }
+            )
+            Spacer(
+                modifier = Modifier.height(dimensionResource(R.dimen.show_screen_spacer_height_3))
+            )
+            Text(
+                text = stringResource(R.string.main_actors),
+                style = MaterialTheme.typography.titleLarge,
+                modifier = Modifier.padding(start = dimensionResource(R.dimen.home_screen_horizontal_padding))
+            )
+            Spacer(
+                modifier = Modifier.height(dimensionResource(R.dimen.show_screen_spacer_height_1))
+            )
+            ShowScreenMainActors(
+                actors = mockMovie.actors,
+                modifier = Modifier
+                    .fillMaxWidth()
+            )
+            Spacer(
+                modifier = Modifier.height(dimensionResource(R.dimen.show_screen_spacer_height_3))
+            )
+            Text(
+                text = setCategoryString(mockMovie.categories.count()),
+                style = MaterialTheme.typography.titleLarge,
+                modifier = Modifier.padding(start = dimensionResource(R.dimen.show_screen_horizontal_padding))
+            )
+            Spacer(
+                modifier = Modifier.height(dimensionResource(R.dimen.show_screen_spacer_height_1))
+            )
+            ShowScreenCategories(
+                categories = mockMovie.categories
+            )
+            Spacer(modifier = Modifier.height(dimensionResource(R.dimen.show_screen_spacer_height_3)))
+            Text(
+                text = stringResource(R.string.recommended),
+                style = MaterialTheme.typography.titleLarge,
+                modifier = Modifier.padding(start = dimensionResource(R.dimen.home_screen_horizontal_padding))
+            )
+            Spacer(
+                modifier = Modifier.height(dimensionResource(R.dimen.show_screen_spacer_height_1))
+            )
+            ShowScreenRecommendedMediaItems(
```

Comment:
You can call this `RecommendedMovies`

Reply:
I named it RecommendedMediaItems because we also display series and soaps